tests/utils/utils.go

```diff
@@ -49,19 +49,20 @@ var (
 func SendTransactionAndWaitForAcceptance(
 	ctx context.Context,
 	wsClient ethclient.Client,
+	rpcClient ethclient.Client,
 	tx *types.Transaction) *types.Receipt {
 
 	newHeads := make(chan *types.Header, 1)
 	subA, err := wsClient.SubscribeNewHead(ctx, newHeads)
 	Expect(err).Should(BeNil())
 	defer subA.Unsubscribe()
 
-	err = wsClient.SendTransaction(ctx, tx)
+	err = rpcClient.SendTransaction(ctx, tx)
```

Comment:
For my own understanding, what's the need to use the `rpcClient` here now when were previously able to use the `wsClient`?

Reply:
So since we have multiple functions where there is a "correct" type of endpoint to use, it might be a good idea to make two type aliases `RPCClient` and `WSClient` that both alias `ethclient.Client`, and then our functions should accept one or the other. Right now you could pass the wrong type of client to any of these functions.

Doesn't have to be done now, I'd be happy to implement this later.

Reply:
My thought was to add `type RPCClient ethclient.Client` and `type WSClient ethclient.Client`, but Go doesn't differentiate those aliases in function arguments. I've instead replaced all function parameters that accept `ethclient.Client` with `SubnetsTestInfo`, and leave it up to function implementation to pick between the RPC and WS clients.